dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtilityTest.java

```diff
@@ -80,9 +81,13 @@ public void testReadBom_path() throws MavenRepositoryException, ArtifactDescript
     if (currentArtifacts.size() != 217) {      
       Set<Artifact> current = Sets.newHashSet(currentArtifacts);
       Set<Artifact> old = Sets.newHashSet(oldArtifacts);
-      SetView<Artifact> added = Sets.difference(current, old);
+      
+      Set<String> currentKeys = current.stream().map(Artifacts::makeKey).collect(Collectors.toSet());
```

Comment:
Don't you want the versions as well? `makeKey()` returns only the `group:artifact`. What about `Artifacts::toCoordinates()`?

Reply:
We don't want the versions, because version updates are expected, and typically number in the dozens so the diff is hard to read. What we care about are truly new or removed dependencies.